src/Scope.ts

```diff
@@ -136,7 +136,7 @@ export class Scope {
     } else if (config) {
       configPubkey = config;
     } else {
-      configPubkey = this._config.configurationAccount;
```

Comment:
let's also remove the default _configs? since there are a bunch of instances new Scope(), which will initialize the default hubble feed

or if we initialize, we initialize based on feeds (klend + hubble), so we don't have to find the oracle price account and load it, etc

Reply:
looking deeper, I see there are more use cases of _config.. for example in getOracleMappingsFromConfig, initialise, etc...

Reply:
Currently _config only includes scope program id + yvaults program id - it will not initialize any feed by default, you will always have to specify it.

We could probably get rid of this as well and default to prod scope/yvaults program id always unless an override is specified in the constructor (and we can remove cluster as well)